app/views/posts/index.html.erb

```diff
@@ -30,4 +30,4 @@
       </div>
     <% end %>
   <% end %>
-</div>
```

Comment:
- [OPTIONAL] When rendering the Post#index page, a `No Method Error` is shown. You have:
`user_post_comment` instead of `user_post_comment_path(...)`. You forgot the `_path`. It would be nice to fix this. 👍 

```ruby
<%= button_to "Delete Comment",
  user_post_comment_path(comment.author_id,
    comment.post_id, comment),
     method: :delete, data: {turbo: false}%>
```  

![image](https://github.com/Fahim-Khan-P/blog/assets/77106962/f60fc367-4934-49fd-bc6f-49c2d0379ea9)